src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.resx

```diff
@@ -4575,4 +4575,4 @@
   <data name="SQL_SettingDeviceFlowWithCredential" xml:space="preserve">
     <value>Cannot use 'Authentication=Active Directory Device Code Flow', if the Credential property has been set.</value>
   </data>
-</root>
+</root>
```

Comment:
Nit: I think we prefer to keep newlines at the end of a file, right?

Reply:
This file gets auto-formatted everytime..

Reply:
Then I'd vote to leave it as auto-formatted to avoid having to revert it manually every time.